Cannonball/SignInViewController.swift

```diff
@@ -65,51 +65,51 @@ class SignInViewController: UIViewController, UIAlertViewDelegate {
                 Crashlytics.sharedInstance().setUserName(session!.userName)
 
                 // Log Answers Custom Event.
-                Answers.logLoginWithMethod("Twitter", success: true, customAttributes: ["User ID": session!.userID])
+                Answers.logLogin(withMethod: "Twitter", success: true, customAttributes: ["User ID": session!.userID])
             } else {
                 // Log Answers Custom Event.
-                Answers.logLoginWithMethod("Twitter", success: false, customAttributes: ["Error": error!.localizedDescription])
+                Answers.logLogin(withMethod: "Twitter", success: false, customAttributes: ["Error": error!.localizedDescription])
             }
         }
     }
 
-    @IBAction func signInWithPhone(sender: UIButton) {
+    @IBAction func signInWithPhone(_ sender: UIButton) {
         // Create a Digits appearance with Cannonball colors.
-        let configuration = DGTAuthenticationConfiguration(accountFields: .DefaultOptionMask)
+        let configuration = DGTAuthenticationConfiguration(accountFields: .defaultOptionMask)
         
-        configuration.appearance = DGTAppearance()
-        configuration.appearance.backgroundColor = UIColor.cannonballBeigeColor()
-        configuration.appearance.accentColor = UIColor.cannonballGreenColor()
+        configuration?.appearance = DGTAppearance()
+        configuration?.appearance.backgroundColor = UIColor.cannonballBeigeColor()
+        configuration?.appearance.accentColor = UIColor.cannonballGreenColor()
 
         // Start the Digits authentication flow with the custom appearance.
-        Digits.sharedInstance().authenticateWithViewController(nil, configuration:configuration) { (session, error) in
+        Digits.sharedInstance().authenticate(with: nil, configuration:configuration!) { (session, error) in
             if session != nil {
                 // Navigate to the main app screen to select a theme.
                 self.navigateToMainAppScreen()
```

Comment:
Should wrap this in `DispatchQueue.main.async { }` to present on the main thread so next screen is shown.

```
DispatchQueue.main.async {
    // Navigate to the main app screen to select a theme.
    self.navigateToMainAppScreen()
}
```